lib/jobi.js

```diff
@@ -147,7 +181,7 @@ class Jobi extends EventEmitter {
 
     // Aggregate and format the log into a writable string
     const log = util.buildLogObject( level, args );
-    const formattedLog = format( log );
+    const formattedLog = this[ withPrefix ]( format( log ) );
```

Comment:
Would rather have the prefix applied to `log.message` rather than the actual output stream. 

Otherwise you break the JSON format by prepending some random string and making the output not valid JSON anymore.